testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/n4idl/tests/helper/N4IDLParseHelper.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.n4idl.tests.helper;
+
+import org.eclipse.n4js.n4JS.Script;
+import org.eclipse.xtext.resource.FileExtensionProvider;
+import org.eclipse.xtext.testing.util.ParseHelper;
+import org.eclipse.xtext.testing.util.ResourceHelper;
+import org.junit.Assert;
+
+import com.google.common.base.Joiner;
+import com.google.inject.Inject;
+
+/**
+ * Replacement for ParseHelper in order to define mode in which file is parsed, also see {@link N4IDLParseHelper}.
+ */
+public class N4IDLParseHelper extends ParseHelper<Script> {
```

Comment:
does it make sense to keep this as a separate class? I'd just add the #parseN4IDL() method to the ordinary `N4JSParseHelper`.

Reply:
To me this was a logical decision, as it follows the pattern of `N4JSXParseHelper`, which is a separate class of similar nature.